### DISCUSSION TITLE
GitHub Copilot checkout with credit card has two submit buttons

### DISCUSSION THREAD

**Discussion by frolic** [community/community#30149] [Copilot Conversations]
<img width="662" alt="image" src="https://user-images.githubusercontent.com/508855/185707375-e628f602-eeff-41e5-9225-5b072db41d33.png">

Note that this is the exact checkout page, not two screenshots stitched together. Above these buttons is the credit card form, but not included in this screenshot because it had PII at the time.

I assume the first is to save the credit card and the second is to process the charge, but it was very unclear and I just happened to guess correctly.

**Comment by radar-blip** [comment]
Odd. Maybe someone should change the wording.
